dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleProtocol.java

```diff
@@ -57,13 +60,22 @@ public class TripleProtocol extends AbstractProtocol {
     private final TriBuiltinService triBuiltinService;
     private final ConnectionManager connectionManager;
     private final String acceptEncodings;
+
+    /**
+     * There is only one
+     */
+    public static boolean CONVERT_NO_LOWER_HEADER = false;
+
     private boolean versionChecked = false;
 
+
     public TripleProtocol(FrameworkModel frameworkModel) {
         this.frameworkModel = frameworkModel;
         this.triBuiltinService = new TriBuiltinService(frameworkModel);
         this.pathResolver = frameworkModel.getExtensionLoader(PathResolver.class)
             .getDefaultExtension();
+        CONVERT_NO_LOWER_HEADER = ConfigurationUtils.getEnvConfiguration(ApplicationModel.defaultModel())
```

Comment:
should not use `ApplicationModel.defaultModel()`

Default model is only used when compatible mode.